packages/url/src/test/index.js

```diff
@@ -723,6 +723,15 @@ describe( 'getQueryArgs', () => {
 				)
 			).toEqual( data );
 		} );
+
+		it( 'should not blow up on malformed params', () => {
+			const url = 'https://andalouses.example/beach?foo=bar&baz=%E0%A4%A';
+
+			expect( getQueryArgs( url ) ).toEqual( {
```

Comment:
Does it also make sense to add a `.not.toThrow()` assertion for this call? That's what the test aims to test if we're looking at the test name.